File: src/Player/PlayerPreset.java
package Player;

import java.util.ArrayList;
import java.util.List;

public class PlayerPreset {
    private String name;
    private Integer attack;

    public PlayerPreset(String name, Integer attack) {
        this.name = name;
        this.attack = attack;
    }

    public String getName() {
        return name;
    }

    public List<Integer> getSkills() {
        return new ArrayList<Integer>();
    }

    public Integer getAttack() {
        return attack;
    }
}

File: src/Player/ValueObject/PlayerPreset.php
<?php

declare(strict_types=1);

namespace Raid\Player\ValueObject;

class PlayerPreset
{
    private $name;

    private $attack;

    private $defence;

    private $maxHealth;

    public function __construct(string $name, int $attack, int $defence, int $maxHealth)
    {
        $this->name = $name;
        $this->attack = $attack;
        $this->defence = $defence;
        $this->maxHealth = $maxHealth;
    }

    /**
     * @return string
     */
    public function getName(): string
    {
        return $this->name;
    }

    /**
     * @return int
     */
    public function getAttack(): int
    {
        return $this->attack;
    }

    /**
     * @return int
     */
    public function getDefence(): int
    {
        return $this->defence;
    }

    /**
     * @return int
     */
    public function getMaxHealth(): int
    {
        return $this->maxHealth;
    }
}
